### PR TITLE
cdc/tests: Add more canal json test

### PR DESCRIPTION
**Pull request by 3AceShowHand**
### What problem does this PR solve? 
add more type info to tests.

### Check List 

Tests 

Code changes

Side effects

Related changes

### Release note 

```release-note
`None`.
```

### CODE REVIEW

File: tests/mq_protocol_tests/cases/case_many_types.go [hunk 1]
```diff
@@ -158,5 +176,32 @@ func (s *ManyTypesCase) Run(ctx *framework.TaskContext) error {
...
+		data["t_mediumint"] = 8388607
+		data["t_int"] = 2147483647
+
+		data["u_tinyint"] = 255
+		data["u_smallint"] = 65535
+		data["u_mediumint"] = 16777215
+		data["u_int"] = 4294967295
+		data["u_bigint"] = uint64(18446744073709551615)
+
+		// data["t_year"] = 2021
```
Comment (overvenus) [inline tests/mq_protocol_tests/cases/case_many_types.go:193]: Could you add some comments?

**Review by overvenus — COMMENTED** [review]
Rest LGTM

**Review by overvenus — APPROVED** [review]

**Review by asddongmen — APPROVED** [review]

**Pull request merged**